ui/shared/bootstrap_overrides.ts

```diff
@@ -1,26 +1,40 @@
 import * as bootstrap from 'bootstrap';
```

Comment:
I rewrote these handlers because they weren't being applied on the SimUI page because the title dropdown loads in via JavaScript. This is more inline with how Bootstrap handlers actually work